joomla/joomla-cms
# Plugin Modal: Apply (save) button + normalize

Pull Request for Issue # .

### Summary of Changes
This PR normalizes the plugin modal as currently used in com_redirect. It also adds a [save] button to the modal footer.


### Testing Instructions
1. normalization:
1.1 disable the plg_redirect
1.2 go to component / Redirect
1.3 in the error message click on the link: "Redirect System Plugin"
[A]
2. save button
2.1 change plugin status in modal to 'enabled'
2.2 change Collect URLs to 'disabled'
2.3 click on [Save] button (the modal should stay open)
2.4 click on [Close] button
[B] 

### Expected result
[A] The modal should open and should be same size / form / shape / etc. as other modals (like edit article in menu single article

[B] after closing the modal, the com_redirect page should refresh and the error message must be replaced with a Notice 'The Redirect Plugin is enabled. The 'Collect URLs' option in the Redirect System Plugin is disabled. Error page URLs will not be collected by this component.'
If you did not change the Collect URLs vanlue to 'disabled' there will be no warning / notice.


### Actual result
[A] modal is small (width and heigth) the other modals
[B] There is no [Save] button, just a [Close]  and [Save & Close] button


### Documentation Changes Required
None
